modules/travel_pay/app/services/travel_pay/service_error.rb

```diff
@@ -22,7 +22,18 @@ def self.raise_mapped_error(error)
         symbolized_body = error.response_body.deep_symbolize_keys
         message = symbolized_body[:message]
       rescue
-        raise Common::Exceptions::ServiceError
+        if error.response_body.nil?
+          Rails.logger.error(
+            message: 'raise_mapped_error received nil response_body. ' \
+                     "status: #{error.response_status}, returning 500. " \
+                     "message: #{error.message}"
+          )
+          raise Common::Exceptions::ExternalServerInternalServerError.new(
```

Comment:
Is there anything `Common::Exceptions::ExternalServerInternalServerError` doesn't provide that `Common::Exceptions::InternalServerError` does?

Reply:
Also, `Common::Exceptions::ExternalServerInteralServerError` is just a `Common::Exceptions::ServiceError`, do we need the if/else behavior?

Reply:
I think I'd just used `Common::Exceptions::ExternalServerInteralServerError` since it was mapped to `500` in our `ERROR_MAP` and I was hoping to maintain some amount of parity between the usual 500's and this "custom" response.
I hadn't seen the inheritance from `ServiceError`, so I'm fine to use one of the other exceptions for some variety here.